### ISSUE TITLE
Failure: integration testkit shutdown

### ISSUE THREAD

**Issue by pvlugter** [lightbend/akkaserverless-javascript-sdk#194]
We sometimes have failures in the sample integration tests, when testkit shutdown is called in the after hook.

For example: https://app.circleci.com/pipelines/github/lightbend/akkaserverless-javascript-sdk/643/workflows/f47f307f-c483-46aa-b33c-1a28ad85cf75/jobs/3614

```
1) Shopping cart service
       "after all" hook for "should remove a cart":
     Uncaught Error: This socket has been ended by the other party
      at Socket.writeAfterFIN [as write] (net.js:457:14)
      at Channel.ondata (internal/streams/readable.js:726:22)
      at addChunk (internal/streams/readable.js:290:12)
      at readableAddChunk (internal/streams/readable.js:265:9)
      at Channel.Readable.push (internal/streams/readable.js:204:10)
      at SSH2Stream.<anonymous> (node_modules/ssh-remote-port-forward/node_modules/ssh2/lib/Channel.js:167:15)
      at parsePacket (node_modules/ssh2-streams/lib/ssh.js:3191:10)
      at SSH2Stream._transform (node_modules/ssh2-streams/lib/ssh.js:701:13)
      at SSH2Stream.Transform._read (internal/streams/transform.js:205:10)
      at SSH2Stream._read (node_modules/ssh2-streams/lib/ssh.js:253:15)
      at SSH2Stream.Transform._write (internal/streams/transform.js:193:12)
      at writeOrBuffer (internal/streams/writable.js:358:12)
      at SSH2Stream.Writable.write (internal/streams/writable.js:303:10)
      at Socket.ondata (internal/streams/readable.js:726:22)
      at addChunk (internal/streams/readable.js:290:12)
      at readableAddChunk (internal/streams/readable.js:265:9)
      at Socket.Readable.push (internal/streams/readable.js:204:10)
      at TCP.onStreamRead (internal/stream_base_commons.js:188:23)
```

**Comment by jroper** [comment]
Possibly fixed in #267.

**Comment by pvlugter** [comment]
Let's close and reopen if needed.
